gulpfile.js

```diff
@@ -53,6 +55,8 @@ gulp.task('flavour', function () {
 var sourceDeps = [
 	'../node_modules/ng-file-upload/dist/ng-file-upload.min.js',
 	'../node_modules/ng-file-upload/dist/ng-file-upload-shim.min.js',
+	'../node_modules/ngmap/build/scripts/ng-map.min.js',
+	'../node_modules/moment/min/moment-with-locales.min.js',
```

Comment:
There's already a script included in index.html with the href of the moment cdn file. Should we remove it from there since it's here?

Reply:
This is for production index.html (/dist) and it's thought for libraries without CDN. 

Maybe it's not a bad idea update the /dist/index.html to include the CDN files instead of adding this dependencies here 🤔